src/views/Publish.vue

```diff
@@ -17,10 +26,18 @@
 <script>
     import { mapGetters } from 'vuex'
     import PrimaryButton from '@components/PrimaryButton.vue'
+    import Navigator from '@components/Navigator.vue'
 
     export default {
         name: 'Publish',
-        components: { PrimaryButton },
+        data: () => {
+            const today = new Date();
+            const date = `${today.getFullYear()}-${today.getMonth()+1}-${today.getDate()}`
+            return {
+                date
```

Comment:
Data is meant for private instance data that the component can read/write. This date field is more of a `computed`